sys/vtimer/vtimer.c

```diff
@@ -286,7 +288,7 @@ static int vtimer_set(vtimer_t *timer)
 
 void vtimer_now(timex_t *out)
 {
-    uint32_t us = HWTIMER_TICKS_TO_US(hwtimer_now() - longterm_tick_start);
+    uint32_t us = HWTIMER_TICKS_TO_US(hwtimer_now()) - longterm_tick_start;
     uint32_t us_per_s = 1000ul * 1000ul;
```

Comment:
Could you change this to `static const uint32_t us_per_s = 1000ul * 1000ul;` while you're poking around here?